### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by Sir-Weebz-A-Lot**

### CODE REVIEW

**Comment by Couls** [comment]
Why are we using string answers instead of numbers?

File: ConsoleApp1/Program.cs [hunk 1]
```diff
@@ -10,134 +10,192 @@ class Program
...
-                Console.ReadLine();
+                }
+                else if (answer == "leave")
+                {
+                    Console.WriteLine(">You leave the firearm and rush to the door, hoping you don't need it");
+                }
+                
+
+
+                Console.WriteLine("");
```
Comment (Couls) [inline ConsoleApp1/Program.cs]: Fix this

File: ConsoleApp1/Program.cs [hunk 2]
```diff
@@ -10,134 +10,192 @@ class Program
...
+                    Console.ReadKey(); // checks for button press
                 }
+
+                Console.WriteLine("We were trapped in here by some... psycho");
+
                 Console.ReadLine();
-                Console.ReadLine();
-                Console.ReadLine();
+                Console.ReadLine(); // For room after text for (input)
+                Console.ReadLine();
```
Comment (Couls) [inline ConsoleApp1/Program.cs]: Use writeline instead of readline for spacing

File: ConsoleApp1/Program.cs [hunk 3]
```diff
@@ -10,134 +10,192 @@ class Program
...
+                }
+                
+
+
+                Console.WriteLine("");
+                Console.WriteLine("(Press any key to continue...)");
+                Console.ReadKey();
+
+                Console.WriteLine("You should be fine after all of this time.");
+                Console.WriteLine(">you stand up and use the bed as a crutch as you try to walk towards the door");
```
Comment (Couls) [inline ConsoleApp1/Program.cs]: Leftover plot? Makes no sense

File: ConsoleApp1/Program.cs [hunk 4]
```diff
@@ -10,134 +10,192 @@ class Program
...
+                                        
+                    if (answer2 == "Shoot")
+                    {
+                        Console.WriteLine();
+                        Console.WriteLine(">You shot all three of them with the weapon you picked up, killing them - you can move on");
+                    }
+                    else if (answer2 == "Don't shoot")
+                    {
+                        Console.WriteLine();
+                        Console.WriteLine(">Your newfound friend shot them all, killing them - you can move on");
```
Comment (Couls) [inline ConsoleApp1/Program.cs]: What friend???

File: ConsoleApp1/Program.cs [hunk 5]
```diff
@@ -10,134 +10,192 @@ class Program
...
-            Console.WriteLine("What's your Name?");
+            Console.WriteLine("Welcome to (unnamed) text adventure!  We hope you enjoy it");
+            Console.WriteLine("Now tell us your name");
             string name;
             name = Console.ReadLine();
             Console.WriteLine();
             Console.WriteLine("Hello, " + name);
             Console.WriteLine();
+            Console.WriteLine(">Adventure start<");
+            Console.WriteLine("(Press enter/space to continue)");
```
Comment (Couls) [inline ConsoleApp1/Program.cs]: Inconsistent change it back to any key

**Review by Couls — CHANGES_REQUESTED** [review]

File: ConsoleApp1/Program.cs (hunk 4 above)
Comment (Sir-Weebz-A-Lot) [reply ConsoleApp1/Program.cs]: The one who woke (user) up.

File: ConsoleApp1/Program.cs (hunk 3 above)
Comment (Sir-Weebz-A-Lot) [reply ConsoleApp1/Program.cs]: Yes, that's all leftover plot.  I'm trying to decide if I should scrap everything there and below it or try to make it of use later in the ta.

File: ConsoleApp1/Program.cs (hunk 5 above)
Comment (Sir-Weebz-A-Lot) [reply ConsoleApp1/Program.cs]: Did you not see what any key does?  I don't want it to look like [a{text}]

File: ConsoleApp1/Program.cs (hunk 1 above)
Comment (Sir-Weebz-A-Lot) [reply ConsoleApp1/Program.cs]: I should have said something but that's my marker between old and new story.  It's fine for now unless we decide to scrap old story.

**Review by Couls — APPROVED** [review]

**Pull request merged**